app/frontend/components/pages/txHistory/transactionHistory.tsx

```diff
@@ -346,5 +347,8 @@ const TransactionHistory = (): h.JSX.Element => {
     </div>
   )
 }
-
-export default TransactionHistory
+// hotfix using memo https://vacuum.atlassian.net/browse/ADLT-1039
+// Send form inputs are updating the global state causing this component
+// to re-render on every input change
+// with long transactionHistory the UX is bad, typing in form is laggy
+export default memo(TransactionHistory)
```

Comment:
as discussed offline, let's research whether/how this "memo" wrapper is actually response to any changes in the TransactionHistory parameters so we clearly understand this wouldn't lead to any inconsistently shown data